packages/superset-ui-demo/storybook/shared/components/createQueryStory.tsx

```diff
@@ -48,11 +48,11 @@ export default function createQueryStory({
                         formData={payload.formData}
                         // @TODO fix typing
                         // all vis's now expect objects but api/v1/ returns an array
-                        queryData={
+                        queriesData={[
                           Array.isArray(payload.queryData)
                             ? payload.queryData[0]
-                            : payload.queryData
-                        }
+                            : payload.queryData,
+                        ]}
```

Comment:
Should we instead do `queriesData={Array.isArray(payload.queryData) ? payload.queryData : [payload.queryData]}`? Or just change `payload.queryData` to `payload.queriesData` and make sure it's always an array.

Reply:
This comes from `ChartDataProvider`, I didn't change there `queryData` to `queriesData` because didn't find who use it, so there it still should be queryData

Reply:
We should probably change this one, too: https://github.com/apache-superset/superset-ui/blob/master/packages/superset-ui-core/src/chart/components/ChartDataProvider.tsx

Reply:
done